home/modules/ROOT/pages/mobile.adoc

```diff
@@ -1,69 +1,91 @@
-= Couchbase Mobile - Embedded Database on the Edge
-:page-layout: landing-page-top-level-sdk
-:page-role: tiles
+= Couchbase Mobile 
+// :page-layout: landing-page-top-level-sdk
+// :page-role: tiles 
+:page-toclevels: 2
 :page-rank: 75
-:!sectids:
 
-include::partial$info-banner.adoc[]
+Couchbase Mobile is a comprehensive NoSQL database solution for building offline-first mobile and edge applications. 
+It enables seamless data synchronization between devices and the cloud, with automatic conflict resolution and security.
 
-= Couchbase Mobile
+== Components
 
-++++
-<div class="card-row">
-++++
+Couchbase Mobile consists of:
 
-[.column]
-====== {empty}
-[.content]
-_Couchbase Mobile_ brings the power of NoSQL to the edge.
-The combination of _Sync Gateway_ and _Couchbase Lite_ coupled with the power of _Couchbase Server_ provides fast, efficient bidirectional synchronization of data between the edge and the cloud.
-Enabling you to deploy your offline-first mobile and embedded applications with greater agility on premises or in any cloud.
+*https://docs.couchbase.com/couchbase-lite/current/index.html[Couchbase Lite]* - Embedded NoSQL database for mobile and edge devices. 
+Couchbase Lite local storage, querying, and offline operations.
 
-[.column]
-====== {empty}
-[.media-left]
-image::secure-manage-and-develop-with-ease.svg[,275]
+*https://docs.couchbase.com/sync-gateway/current/introduction.html[Sync Gateway]* - Secure, high-performance synchronization server for cloud-to-edge data sync. 
+Provides authentication, access control, and RESTful APIs. 
+Both on-premises and Capella (cloud) deployments use Sync Gateway.
 
-++++
-</div>
-++++
+*https://docs.couchbase.com/couchbase-edge-server/current/introduction/intro.html[Edge Server]* - Lightweight database for edge/IoT environments. 
+Edge Server enables local data processing and synchronization with Couchbase Lite and Capella.
 
-== Embedded Database on the Edge
-++++
-<div class="card-row two-column-row">
-++++
+*https://docs.couchbase.com/home/cloud.html[Capella]* - Fully managed Database-as-a-Service (DBaaS) for Couchbase, including managed mobile sync via App Services.
 
-[.column]
-.Couchbase Lite
+*https://docs.couchbase.com/cloud/app-services/references/rest-api-introduction.html[App Services]* - The managed synchronization layer in Capella, providing the same sync capabilities as Sync Gateway but fully managed in the cloud.
```

Comment:
Feels odd to include Capella as a component here when we can just specify App Services and Capella usage is implied.

It's the same as Sync Gateway implicitly requiring Couchbase Server for storage.